gorjana/flows/products/README.md

```diff
@@ -1,6 +1,19 @@
-# Products
+---
+description: >-
+  Delve into gorjana's product management strategies and learn how to optimize SKU
+  settings for seamless integration with Shopify.
+---
 
-Out of the box Shopify product sync
-Out of the box NetSuite product sync
+# Product
 
-(link to HotWax shopify and netsuite product doc)
\ No newline at end of file
+gorjana's uses `SKU` as their primary product identifier. Before syncing products from Shopify, make sure to change this settings in the Product Store.
```

Comment:
gorjana uses 'Shopify Product SKU' as their primary product identifier. Before syncing products from Shopify, make sure to change this setting in the Product Store.